client/src/containers/SignUp/SignUp.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom'
+import SmallContainer from '../../components/SmallContainer/SmallContainer';
+import InputField from '../../components/InputField/InputField';
+import { getError, hasError, isTouched } from '../../libs/validate';
+import handleErrors from './Schema';
+import './SignUp.css';
+
+const SignUp = () => {
+  const [ user, setUser ] = useState({ name: '', email: '', password: '' });
+  const [ touched, setTouched ]= useState({});
+  const [ error, setError ] = useState({});
+  
+  const touchedFields = {}
+  
+  const handleChange = field => event => {
+    setUser({ ...user, [field]: event.target.value }, () => handleErrors(user, setError));
```

Comment:
What's the use case of the second argument to setUser?

Reply:
I was trying to use that as a callback, but during the hackathon, I found that the "useEffect" is used to define callbacks using dependencies. Plus the callback is not needed here, I'll just change it.